packages/connect/src/ui.ts

```diff
@@ -0,0 +1,41 @@
+import { authenticate, AuthOptions, FinishedData } from './auth';
+import { defineCustomElements } from '@blockstack/connect-ui';
+
+export const showBlockstackConnect = (authOptions: AuthOptions) => {
+  defineCustomElements();
+  const element = document.createElement('connect-modal');
+  element.authOptions = authOptions;
+  document.body.appendChild(element);
+  const finishedWrapper = (finishedData: FinishedData) => {
+    element.remove();
+    const callback = authOptions.onFinish || authOptions.finished;
+    if (callback) {
+      callback(finishedData);
+    }
+  };
+  element.addEventListener('signUp', () => {
```

Comment:
These are custom events I take it, convention for native browser events is all lower case, any value keeping to that convention? 🤔

Reply:
Do we need a prefix? Could this overlap with any other page events?